b2sdk/api.py

```diff
@@ -209,13 +209,13 @@ def authorize_automatically(self):
         return self.session.authorize_automatically()
 
     @limit_trace_arguments(only=('self', 'realm'))
-    def authorize_account(self, realm, application_key_id, application_key):
+    def authorize_account(self, application_key_id, application_key, realm='production'):
```

Comment:
I was considering asking for providing remaining `realm` options as e.g. `Literal` type or in the `docstring`. But these additional options are only for B2 developers, right? So it probably wouldn't make sense to add it here :/